.gitignore

```diff
@@ -5,6 +5,9 @@
 gae_runtime/*
 third_party/*
 core/templates/prod/*
+static/fonts/*
```

Comment:
In static/fonts goes all font-awesome and bootstrap fonts when concatenation happens.
One thing I was not sure. Does third_party library gets uploaded during deployment? I have putt all fonts and generated css in third_party/generated/ . Where do you think should the fonts go? and I am afraid to put third_party.css in dev/head/css because i do not want it to be uploaded during push.